render/wgpu/Cargo.toml

```diff
@@ -21,7 +21,7 @@ image = { version = "0.24.5", default-features = false }
 ouroboros = "0.15.5"
 typed-arena = "2.0.1"
 once_cell = "1.17.0"
-gc-arena = { git = "https://github.com/ruffle-rs/gc-arena" }
+gc-arena = { git = "https://github.com/kyren/gc-arena", rev = "3195fbed9f2ba1683fb113e3d7d84794fe3465d5" }
```

Comment:
Can you make this a workspace dependency:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace

Reply:
TIL, I didn't know that was possible.

Reply:
It's a pretty new feature, but we're using the latest stable Rust, so we can use it :)